CHANGELOG.asciidoc

```diff
@@ -3,6 +3,55 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-8.11.2]]
+=== Beats version 8.11.2
+https://github.com/elastic/beats/compare/v8.11.1\...v8.11.2[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+- Elastic-agent-autodiscover library updated to version 0.6.4, disabling metadata for deployment and cronjob. Pods that will be created from deployments or cronjobs will not have the extra metadata field for kubernetes.deployment or kubernetes.cronjob, respectively. {pull}36877[36877]
+- Defaults are changing for some options in the queue and Elasticsearch output, to improve typical performance based on current benchmark data. All changes can be overridden by setting them explicitly in the beat configuration. {pull}36990[36990] The changes are:
```

Comment:
```suggestion
```

This was from a merge conflict https://github.com/elastic/beats/commit/d1691c490ed83390d19ed75a38d6fdd958561671